pop-api/examples/fungibles/README.md

```diff
@@ -0,0 +1,47 @@
+# PSP22 Fungible Token with Pop API
+
+PSP22 is a fungible token standard for WebAssembly smart contracts running on blockchains based on the [Substrate][substrate] framework. It is an equivalent of Ethereum's [ERC-20][erc20]. The definition of the PSP22 standard can be found [here][psp22].
+
+This repository contains a simple, minimal implementation of the PSP22 token in [ink!][ink] smart contract programming language (EDSL based on Rust), utilizing the [Pop API Fungibles][pop-api-fungibles] feature.
```

Comment:
Way too much detail, also we don't want to repeat the docs in the pop api about the psp22 contract. Just explain the contract in a simple way :), no need to make it fancy or anything.

"This contract shows a contract that allows interaction and management of a fungible token following the psp22 standard. In this specific contract only the contract owner has a permission to call specific methods. This is because ....
This is only one way of how the fungibles api can be utilised, other ways could be ...

Tests are written with pop drink ... and can be run with `cargo test --release`

## Support

Be part of our passionate community of Web3 builders. [Join our Telegram](https://t.me/onpopio)!

Feel free to raise issues if anything is unclear, have ideas or want to contribute to Pop! We would be very happy to see other examples utilising the fungibles API!

For any questions related to ink! you can also go to [Polkadot Stack Exchange](https://polkadot.stackexchange.com/) or
ask the [ink! community](https://t.me/inkathon/1).

Reply:
The level of details in the current `README.md` is not that much, and I think we should give them some details and reference link for them to learn more. For example:

> Tests are written with pop drink ... and can be run with cargo test --release

Developer will have a concern what is pop-drink and why they need to use it instead of ink e2e test. So at least let them know that we need pop-drink because pop-api...then reference link